setup.sh

```diff
@@ -2,73 +2,37 @@
 
 set -e
 
-SCRIPT_ROOT=$(cd "$(dirname "$0")"; pwd -P)
-PATH_TO_SWIFT_TOOLCHAIN="$SCRIPT_ROOT/swift-flowkey.xctoolchain"
-UNAME=`uname`
+readonly SCRIPT_ROOT=$(cd "$(dirname "$0")"; pwd -P)
+readonly SWIFT_ANDROID_TOOLCHAIN_PATH="${SWIFT_ANDROID_TOOLCHAIN_PATH:-$SCRIPT_ROOT/swift-android.xctoolchain}"
 
 log() {
     echo "[swift-android-toolchain] $*"
 }
 
-if [[ ! ${ANDROID_NDK_PATH} ]] || [[ ! `cat "$ANDROID_NDK_PATH/changelog.md" | grep "r16b"` ]]; then
-    log "Please define ANDROID_NDK_PATH and point to your local version of ndk-r16b"
-    if [[ ${UNAME} == "Darwin" ]]; then
-        log "Download from https://dl.google.com/android/repository/android-ndk-r16b-darwin-x86_64.zip"
-    elif [[ ${UNAME} == "Linux" ]]; then
-        log "Download from https://dl.google.com/android/repository/android-ndk-r16b-linux-x86_64.zip"
-    fi
-    exit 1
-fi
-
 clean() {
-    git -C $SCRIPT_ROOT clean -xdf
+    rm -rf $SCRIPT_ROOT/temp $SCRIPT_ROOT/swift-android.xctoolchain
 }
 
 rm -rf $SCRIPT_ROOT/temp
 mkdir -p $SCRIPT_ROOT/temp
 cd $SCRIPT_ROOT/temp
 
-downloadArtifacts() {
+downloadToolchain() {
     mkdir -p $SCRIPT_ROOT/temp
     cd $SCRIPT_ROOT/temp
 
-    BASEPATH="https://swift-toolchain-artifacts.flowkeycdn.com"
+    log "Downloading Toolchain..."
 
-    if [[ ${UNAME} == "Darwin" ]]; then
-        VERSION="20200112.01"
-    elif [[ ${UNAME} == "Linux" ]]; then
-        VERSION="stable_2"
-    fi
+    # mirror of https://github.com/vgorloff/swift-everywhere-toolchain/releases/download/1.0.68/swift-android-toolchain.tar.gz
+    TOOLCHAIN_PATH="https://swift-toolchain-artifacts.flowkeycdn.com/swift-android-toolchain-5.3.3.tar.gz"
```

Comment:
our toolchain is just a thin wrapper around the great work of vgorloff. we should add some reference to the readme

Reply:
is this mac only? i think we should make that clear in the readme too if so

Reply:
seehttps://github.com/flowkey/swift-android-toolchain/pull/12/files#r665360193